jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -271,14 +272,23 @@ private static DiagnosticProperties doDump(String diagnosticCommand) {
 			}
 			if (!dumpType.isEmpty()) {
 				String fileDirective = ("system".equals(dumpType) && IPC.isZOS) ? ":dsn=" : ":file="; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
-
-				String request = dumpType + fileDirective + parts[1];
-				try {
-					String actualDumpFile = triggerDumpsImpl(request, dumpType + "DumpToFile"); //$NON-NLS-1$
-					result = DiagnosticProperties.makeStringResult("Dump written to " + actualDumpFile); //$NON-NLS-1$
-				} catch (InvalidDumpOptionExceptionBase e) {
-					IPC.logMessage("doDump exception: ", e.getMessage()); //$NON-NLS-1$
-					DiagnosticProperties.makeExceptionProperties(e);
+				if (parts.length == 1) {
+					// there is no file path/name specified, use default settings like -Xdump
+					int rc = triggerDefaultDumpsImpl(dumpType);
```

Comment:
Any reason the existing native can't be used here, it looks like it would work.

Reply:
The existing native `openj9.internal.tools.attach.target.DiagnosticUtils.triggerDumpsImpl()` is a wrapper of `com.ibm.jvm.Dump.triggerDumpsImpl()` which lives along with `com.ibm.jvm.Dump.JavaDumpImpl()`, `com.ibm.jvm.Dump.SnapDumpImpl()`, `com.ibm.jvm.Dump.SystemDumpImpl()`, and `com.ibm.jvm.Dump.HeapDumpImpl()`. Those `Java/Snap/System/Heap` specific dump native don't provide path/name, and they are in separated natives to avoid path/name parsing within native `Dump.triggerDumpsImpl()`. 
Similarly `triggerDefaultDumpsImpl()` doesn't invoke `Dump.triggerDumpsImpl()` directly either.

Reply:
I believe triggerDumpsImpl can be used. As long as `":file=" + parts[1]` isn't specified, it will use the default names. Please give this a try. Otherwise there are more changes required to triggerDefaultDumpsImpl which I'll call out in further review.

Reply:
Thanks, `triggerDumpsImpl()` did the trick when `:file=name` isn't supplied in the dump request string. Updated the PR, please have another look.

`com.ibm.jvm.Dump.JavaDumpImpl()`, `com.ibm.jvm.Dump.SnapDumpImpl()`, `com.ibm.jvm.Dump.SystemDumpImpl()`, and `com.ibm.jvm.Dump.HeapDumpImpl()` should be able to use `Dump.triggerDumpsImpl()` as well, but I am going to leave it to another PR.